eth/handler.go

```diff
@@ -712,8 +709,8 @@ func (pm *ProtocolManager) handleMsg(p *peer) error {
 
 		// Mark the peer as owning the block and schedule it for import
 		p.MarkBlock(request.Block.Hash())
+		// Send to fetcher (which internally filters duplicates)
```

Comment:
duplicates -> deduplicates? :)

Reply:
Well, it filters the duplicates. As in, "this is a duplicate of another, and those two over there are also duplicates".